psd-web/app/helpers/investigations_helper.rb

```diff
@@ -248,6 +248,34 @@ def suggested_previous_owners
     all_past_owners || []
   end
 
+  def source_details_rows(investigation, viewing_user)
+    rows = [
+      investigation.should_display_date_received? ? { key: { text: "Received date" }, value: { text: investigation.date_received.to_s(:govuk) } } : nil,
+      investigation.should_display_received_by? ? { key: { text: "Received by" }, value: { text: investigation.received_type.upcase_first } } : nil,
+      { key: { text: "Source type" }, value: { text: investigation.complainant.complainant_type } },
+      { key: { text: "Contact details" }, value: { text: investigation.contact_details_for_display(viewing_user) } }
+    ]
+
+    rows.compact!
+    rows
+  end
+
+  # rubocop:disable Rails/OutputSafety
+  def team_list_html(investigation)
```

Comment:
Not sure we need the `_html` suffix here, as most of the helpers generate HTML.

```suggestion
  def list_of_teams_with_access(investigation)
```